crates/e2e/src/xts.rs

```diff
@@ -176,14 +177,10 @@ pub struct ContractsApi<C: subxt::Config, E: Environment> {
 impl<C, E> ContractsApi<C, E>
 where
     C: subxt::Config,
-    C::AccountId: Into<C::Address> + serde::de::DeserializeOwned,
-    <C::ExtrinsicParams as ExtrinsicParams<C::Index, C::Hash>>::OtherParams: Default,
-
+    C::AccountId: serde::de::DeserializeOwned,
+    C::AccountId: scale::Codec,
```

Comment:
I'm not sure why they removed the `Parameter` (which is `Codec + Clone` etc. constraint from `type AccountId`

Reply:
@jsdw I meant to ping you about this last week, but yeah this shouldn't have been removed imo

Reply:
I spoke with @jsdw about this and the outcome was that all bounds which are not required within `subxt` itself are removed. And that if we want glbla bounds we should use our own local trait.

https://github.com/paritytech/subxt/pull/804